ding_periodical.module

```diff
@@ -48,7 +48,9 @@ function ding_periodical_menu() {
  * @return array
  *   Render array with Ajax commands.
  */
-function ding_periodical_reserve_ajax($entity, $local_id, $volume, $issue) {
+function ding_periodical_reserve_ajax($entity, $periodical) {
+  $periodical_id = base64_decode($periodical);
+  list($local_id, $volume, $issue) = explode('|', $periodical_id);
```

Comment:
If we are to change the signature of the callback please update the DocBlock accordingly.